integrations/standalone/tests/integration/dataclass-editor-business-dataclass.spec.ts

```diff
@@ -1,12 +1,25 @@
 import { expect, test } from '@playwright/test';
 import { DataClassEditor } from '../pageobjects/DataClassEditor';
 
-test('load data', async ({ page }) => {
-  const editor = await DataClassEditor.openDataClass(page, 'dataclasses/dataclass/DataClass.d.json');
+let editor: DataClassEditor;
+
+test.beforeEach(async ({ page }) => {
+  editor = await DataClassEditor.openDataClass(page, 'dataclasses/dataclass/DataClass.d.json');
+});
+
+test('master header', async () => {
+  await expect(editor.title).toHaveText('Business Data Editor');
+});
+
+test('detail header', async () => {
+  await expect(editor.detail.title).toHaveText('Business Data - DataClass');
+});
```

Comment:
just keep in mind, that playwright tests took long to run, especially every startup of one test will took some time and give pressure on the engine.
So maybe sometimes it is better to combine some stuff in one test, e.g. in this case check all header titles